Invalid export in /googleapis/build/src/index.d.ts

Following error raises when the library is used with [Loopback 4](https://loopback.io/doc/en/lb4/#overview)

`node_modules/googleapis/build/src/index.d.ts:5:10 - error TS1005: 'from' expected.

5 export * as Common from 'googleapis-common';
           ~~

node_modules/googleapis/build/src/index.d.ts:5:13 - error TS1005: ';' expected.

5 export * as Common from 'googleapis-common';
              ~~~~~~

node_modules/googleapis/build/src/index.d.ts:5:20 - error TS1005: ';' expected.

5 export * as Common from 'googleapis-common';
                     ~~~~

node_modules/googleapis/build/src/index.d.ts:5:25 - error TS1005: ';' expected.

5 export * as Common from 'googleapis-common';
`


#### Environment details
##### 1.
  - OS: Windows 10 x64
  - Node.js version: 12.13.1
  - npm version: 6.12.1
  - `googleapis` version: 59.0.0
  - `typescript` version: 3.7.5

##### 2.
  - OS: Ubuntu 18.04 LTS
  - Node.js version: 12.10.0
  - npm version: 6.14.4
  - `googleapis` version: 59.0.0
  - `typescript` version: 3.7.5

Thanks!

--- a/src/index.ts
+++ b/src/index.ts
@@ -15,8 +15,9 @@
 import {GoogleApis} from './googleapis';
 const google = new GoogleApis();
 export {google, GoogleApis};
-export * as Common from 'googleapis-common';
-export * as Auth from 'google-auth-library';
+import * as Common from 'googleapis-common';
+import * as Auth from 'google-auth-library';
+export {Common, Auth};
 
 export {abusiveexperiencereport_v1} from './apis/abusiveexperiencereport/v1';
 export {acceleratedmobilepageurl_v1} from './apis/acceleratedmobilepageurl/v1';

